homeassistant/components/fritz/strings.json

```diff
@@ -0,0 +1,43 @@
+{
+  "config": {
+    "flow_title": "FRITZ!Box Tools: {name}",
+    "step": {
+      "confirm": {
+        "title": "Setup FRITZ!Box Tools",
+        "description": "Discovered FRITZ!Box: {name}\n\nSetup FRITZ!Box Tools to control your {name}",
+        "data": {
+          "username": "Username",
```

Comment:
Please use `key:common` here and the other data places

Reply:
Fixed